github/billing.go

```diff
@@ -87,6 +106,20 @@ func (s *BillingService) GetStorageBillingOrg(ctx context.Context, org string) (
 	return storageOrgBilling, resp, err
 }
 
+// GetAdvancedSecurityActiveCommittersOrg returns the GitHub Advanced Security active committers for an organization per repository.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-github-advanced-security-active-committers-for-an-organization
+func (s *BillingService) GetAdvancedSecurityActiveCommittersOrg(ctx context.Context, org string) (*ActiveCommitters, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/settings/billing/advanced-security", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	ActiveOrgCommitters := new(ActiveCommitters)
```

Comment:
I'm realizing now that I did not do a very thorough job of reviewing #1797, and therefore this `billing.go` does not match the style for the rest of this repo.

I'd like to fix that, and maybe you can assist me.  If you could please apply these modifications to all the endpoints within `billing.go` (and the third change to all the endpoints in `billing_test.go`), that would be greatly appreciated... and please do not use `force-push` if possible to make the reviews much easier.

So first, after line 117, please add a blank line.